charts/azuremonitor-containers/templates/ama-logs-deployment.yaml

```diff
@@ -182,6 +185,23 @@ spec:
    {{- end }}
    {{- end }}
    volumes:
+    - name: kube-api-access
+      projected:
+        sources:
+        - serviceAccountToken:
+            path: token
+            expirationSeconds: 3600
+        - configMap:
+            items:
+            - key: ca.crt
+              path: ca.crt
+            name: kube-root-ca.crt
+        - downwardAPI:
+            items:
+            - fieldRef:
```

Comment:
For my understanding, why do we need the namespace ?

Reply:
> For my understanding, why do we need the namespace ?
> We dont use namespace. I kept the same behavior as today i.e.  under /var/run/secrets/kubernetes.io/serviceaccount, there will be token, crt files and namespace. The namespace here indicates what the namespace of the current running pod.